index.html

```diff
@@ -46,127 +42,75 @@ <h1>Cakes for any event<br />
 											<p>A free and fully responsive site template</p>
 										</header>
 										<p>Aenean ornare velit lacus, ac varius enim ullamcorper eu. Proin aliquam facilisis ante interdum congue. Integer mollis, nisl amet convallis, porttitor magna ullamcorper, amet egestas mauris. Ut magna finibus nisi nec lacinia. Nam maximus erat id euismod egestas. Pellentesque sapien ac quam. Lorem ipsum dolor sit nullam.</p>
-										<ul class="actions">
+										<!-- <ul class="actions">
 											<li><a href="orders.html" class="button big">Order Now</a></li>
```

Comment:
I don't think this button is necessary any more but wanted to flag it in case you think it might be helpful.

Reply:
yea i am not sure where this button would redirect to now that we don't have a singular order page. Maybe the "About" page?

Reply:
That's a good idea, I just added the button back linking to the "About" page.